README.md

```diff
@@ -45,15 +45,33 @@ $ make
+```
```

Comment:
This JSON is in a single line - it may be hard to read. Please consider formatting this JSON.